bitex/interface/bithumb.py

```diff
@@ -0,0 +1,103 @@
+"""Bitstamp Interface class."""
+# Import Built-Ins
+import logging
+from bitex.api.REST.bithumb import BithumbREST
+from bitex.interface.rest import RESTInterface
+from bitex.utils import check_and_format_pair, format_with
+from bitex.formatters import BithumbFormattedResponse
+
+# Init Logging Facilities
+log = logging.getLogger(__name__)
+
+
+class Bithumb(RESTInterface):
+    """Bithumb REST API Interface Class."""
+
+    def __init__(self, **api_kwargs):
+        """Initialize Interface class instance."""
+        super(Bithumb, self).__init__('Bithumb', BithumbREST(**api_kwargs))
+
+    def request(self, endpoint, authenticate=False, **req_kwargs):
+        """Generate a request to the API."""
+        if not authenticate:
+            return super(Bithumb, self).request('GET', endpoint, authenticate=authenticate,
+                                                **req_kwargs)
+        return super(Bithumb, self).request('POST', endpoint, authenticate=authenticate,
+                                            **req_kwargs)
+
+    def _get_supported_pairs(self):
+        """Return a list of supported pairs."""
+        pairs = ['BTC', 'ETH', 'DASH', 'LTC', 'ETC', 'XRP', 'BCH', 'XMR', 'ZEC', 'QTUM', 'BTG',
+                 'EOS']
```

Comment:
is there no symbols endpoint on bithumb?

Reply:
No, many korea exchange there's no symbols endpoint